packages/common/src/services/remote-config/remote-config.ts

```diff
@@ -74,7 +74,7 @@ export const remoteConfig = <
   let client: Client | undefined
 
   const emitter = new EventEmitter()
-  emitter.setMaxListeners(20)
+  emitter.setMaxListeners(100)
```

Comment:
This for the error you were noticing earlier?

Reply:
Yessir. Thanks to @sliptype for the tip

Reply:
woah interesting, why do they even have a max?

Reply:
I _think_ it's to warn you if you're creating tons of them in a loop? We are creating one per component that calls `useFeatureFlag` so it's kinda in a loop, but we also know that we're not making 1000s of them given other limitations on the size of UI lists.